tests/dummy/config/environment.js

```diff
@@ -43,6 +43,8 @@ module.exports = function(environment) {
 
     apiHost: 'http://localhost:4200',
 
+    googleClientID: '271351926386-jdh86lc996s5r6kia2dk2r7edrsmke85.apps.googleusercontent.com',
```

Comment:
Here's the Client ID of the ESA example app: 694766332436-1g5bakjoo5flkfpv3t2mfsch9ghg7ggd.apps.googleusercontent.com